v1.9/kubeadm/README.md

```diff
@@ -0,0 +1,99 @@
+# kubeadm
+Official documentation:
+ - https://kubernetes.io/docs/setup/independent/install-kubeadm/
+ - https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/
+
+
+# Reproduce Conformance Results:
+
+## pre-reqs
+- vagrant + virtualbox
```

Comment:
This isn't a mandate, just a provable way of reproducing these results.

Vagrant is a good MVP for this since it provides code for doing so.
I ran into two issues when trying to reproduce that this code addresses:
- needed to disable swap
- not enough RAM to schedule conformance pod

Readers of this doc should be able to copy the shell script portion of the `Vagrantfile` to fit their needs.